src/status_im/subs/wallet/wallet.cljs

```diff
@@ -376,11 +386,13 @@
  :<- [:wallet/accounts]
  :<- [:profile/currency]
  :<- [:wallet/selected-networks->chain-ids]
- (fn [[accounts currency chain-ids]]
+ :<- [:wallet/prices-per-token]
+ (fn [[accounts currency chain-ids prices-per-token]]
    (zipmap (map :address accounts)
-           (map #(utils/calculate-balance-from-tokens {:currency  currency
-                                                       :tokens    (:tokens %)
-                                                       :chain-ids chain-ids})
+           (map #(utils/calculate-balance-from-tokens {:currency         currency
```

Comment:
`[Not a suggestion for change in this PR]` This is a good example where we are pulling too much in subs and the code is inefficient.

To compute this sub we only need the addresses and tokens of accounts. The result of `:wallet/accounts` sub contains lots of unrelated data, including timestamp and clock, things that can change more often and which would invalidate the cache. The better thing to do is to make the inputs be 2 subs, the accounts addresses and accounts tokens, not the whole `:wallet/accounts`.

This `:wallet/accounts` sub reminds me of the `:communities/community` sub, which if we subscribe to it will be guaranteed inefficient because it can change for too many reasons (thus changes more frequently and invalidates caches and causes re-processing by Reagent more easily/frequently).

https://github.com/status-im/status-mobile/blob/c6c3e089caf7f18a2a4f3bf3df3890513b0fd1aa/src/status_im/subs/communities.cljs#L21-L28